miio/__init__.py

```diff
@@ -3,9 +3,10 @@
 from miio.vacuumcontainers import (VacuumStatus, ConsumableStatus, DNDStatus,
                                    CleaningDetails, CleaningSummary, Timer)
 from miio.vacuum import Vacuum, VacuumException
-from miio.plug import Plug
-from miio.plug_v1 import PlugV1
-from miio.plug_v3 import PlugV3
+from miio.chuangmi_plug import Plug
+from miio.chuangmi_plug import PlugV1
+from miio.chuangmi_plug import PlugV3
+from miio.chuangmi_plug import ChuangmiPlug
```

Comment:
`from miio.chuangmi_plug import (Plug, PlugV1, PlugV3, ChuangiPlug)` would be nicer here.